scripts/compute-tags.sh

```diff
@@ -36,4 +36,4 @@ fi
 
 IMAGE_TAGS="${IMAGE_TAGS} --tag docker.io/${SNAPSHOT_TAG} --tag quay.io/${SNAPSHOT_TAG}"
 
-echo ${IMAGE_TAGS}
+echo "${IMAGE_TAGS}"
```

Comment:
```suggestion
echo "${IMAGE_TAGS[@]}"
```
otherwise it only prints the first item